src/site/content/en/accessible/accessible.11tydata.js

```diff
@@ -34,6 +34,15 @@ module.exports = {
           'labels-and-text-alternatives',
         ],
       },
+      {
+        title: 'Understand how your design and CSS can support users with different needs.',
```

Comment:
@rachelandrew the linter is saying to move the value of this property to a newline `⏎` and then indent 10 spaces

Reply:
You might also try a couple of options:

- Run eslint with the fix flag either using `npm run lint:js -- --fix` or `npx eslint . --fix`
- Or, depending on your editor, you could install the ESLint extension and set it to `fixOnSave`.

Reply:
yep running eslint locally, that's why I'm puzzled! (in VSCode)